samples/cpp/tutorial_code/video/bg_sub.cpp

```diff
@@ -98,7 +98,7 @@ void processVideo(char* videoFilename) {
         exit(EXIT_FAILURE);
     }
     //read input data. ESC or 'q' for quitting
-    while( (char)keyboard != 'q' && (char)keyboard != 27 ){
+    while( keyboard != 'q' && keyboard != 27 ){
```

Comment:
Suppose it doesn't make a big difference on execution but IMO it would be better to initialize keyboard prior to check. Just in case